gps-nmea/gps_nmea_i2c.go

```diff
@@ -0,0 +1,359 @@
+// Package gpsnmea implements an NMEA serial gps for a gpsrtki2cnonetwork.
+package nmea
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/d2r2/go-i2c"
```

Comment:
how is this package doing? Do we like it? 

Might be good to find a template for the eventual changing of i2c sensors in rdk.

Reply:
Yeah works great! only annoying thing is that it logs every read/write so I had to add a line to reduce the number of log messages every time I created a connection.